templates/usr/local/bin/haproxy-ocsp-deploy.j2

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+#
+# Ansible managed
+#
+# set -x;
+set -e;
+set -o pipefail;
+#
+thisFile="$(readlink -f "${0}")";
+thisFilePath="$(dirname "${thisFile}")";
+
+for path in $(ls -1d {{ haproxy_scripts_ssl_src_path }}/*/); do
+  cert="$(basename $path)";
+
+  prefix="100";
+  removePrefix="000";
+
+  if [ $cert == "{{ haproxy_scripts_ssl_first_cert }}" ]; then
+    prefix="000";
+    removePrefix="100";
+  fi
+
+  ocspUrl=$(openssl x509 -noout -ocsp_uri -in ${path}{{ haproxy_scripts_ssl_cert_name }});
```

Comment:
ocspUrl="$(openssl x509 -noout -ocsp_uri -in ${path}{{ haproxy_scripts_ssl_cert_name }})";